webrtc.html

```diff
@@ -8064,8 +8064,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   <code><a data-for="RTCDataChannel">negotiated</a></code>,
                   <code><a data-for="RTCDataChannel">id</a></code> and
                   <code><a data-for="RTCDataChannel">priority</a></code> attributes
-                  to the values of their corresponding dictionary members (if
-                  present in the dictionary).</p>
+                  to the values of their corresponding dictionary members, if
+                  present in the dictionary. For each member not present in the
+                  dictionary, initialize the corresponding attribute to its
+                  default value, as described in the attribute's description.
```

Comment:
Good suggestion. I haven't cleaned up the attribute descriptions yet, but I at added default `null` values for the `RTCDataChannelInit` members and simplified this step.